ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -440,22 +455,28 @@ bool solveOneTimeStepOneProcess(const unsigned process_index,
 
     auto const post_iteration_callback = [&](unsigned iteration,
                                              GlobalVector const& x) {
-        output_control.doOutputNonlinearIteration(
-            process, process_index, process_data.process_output, timestep, t,
-            x, iteration);
+        output_control.doOutputNonlinearIteration(process, process_id,
+                                                  process_data.process_output,
+                                                  timestep, t, x, iteration);
     };
 
     bool nonlinear_solver_succeeded =
         nonlinear_solver.solve(x, post_iteration_callback);
 
+    if (nonlinear_solver_succeeded)
+    {
+        process.postNonLinearSolver(x, t, process_id);
```

Comment:
Please add the info about this function being executed only on nonlinear solver success to the `postNonlinearSolver` member documentation.

Reply:
Modified the documentation of postNonLinearSolver as
```
    /// Calculates secondary variables, e.g. stress and strain for deformation
    /// analysis, only after nonlinear solver being successfully conducted.
```